tasks/libs/copyright.py

```diff
@@ -29,9 +29,15 @@
     '/pkg/security/probe/custom_events_easyjson.go',
     '/pkg/security/probe/serializers_easyjson.go',
     '/pkg/security/secl/model/accessors.go',
+    '/pkg/proto/msgpgo/.*_gen(_test){,1}.go',
+    '/pkg/trace/pb/.*_gen(_test){,1}.go',
+    '/pkg/security/probe/fields_resolver.go',
     # These are files that we should not add our copyright to
     '/third_party/golang/',
     '/third_party/kubernetes/',
+    '/patch/grpc-go-insecure/',
```

Comment:
Did you mean this instead?
```suggestion
    '/internal/patch/grpc-go-insecure/',
```

Reply:
Both work but as it a simple regex. For consistency I'll update it to have the full package path.

Reply:
Right, didn't realize it was a regex :) both make sense then!